### PR TITLE
python: Support http/1.1

### PR DESCRIPTION
**Pull request by corona10**
ISSUE: https://github.com/rust-kr/browser-engineering/issues/14

### CODE REVIEW

File: python/http.py [hunk 1]
```diff
@@ -42,9 +42,12 @@ def _get_headers_and_body(sock, host, port, path):
     sock.connect((host, port))
 
     # 5. Send request
-    sock.send(f"GET {path} HTTP/1.0\r\n".encode())
+    sock.send(f"GET {path} HTTP/1.1\r\n".encode())
     sock.send(f"Host: {host}\r\n".encode())
-    sock.send("Accept-Encoding: br,gzip,deflate\r\n".encode())
+    sock.send(f"Connection: close\r\n".encode())
+    sock.send(f"User-Agent: homemade-browser\r\n".encode())
```
Comment (corona10) [inline python/http.py]: Any idea about naming user-agent?

Comment (youknowone) [reply python/http.py]: something prefixed by `Mozilla/5.0`

File: python/http.py [hunk 2]
```diff
@@ -40,11 +40,18 @@ def request(url):
 
 def _get_headers_and_body(sock, host, port, path):
     sock.connect((host, port))
+    brotli = None
```
Comment (youknowone) [inline python/http.py]: is this intended change?

Comment (corona10) [reply python/http.py]: ah sorry for testing

**Pull request closed without merging**